<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlockScript v0.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BlockScript.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入代码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="maximumSize">
       <size>
        <width>155</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>68</width>
         <height>450</height>
        </rect>
       </property>
       <attribute name="label">
        <string>基础</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="ActiveLabel" name="label_create_start">
          <property name="text">
           <string>起始块</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ActiveLabel" name="label_create_proc">
          <property name="text">
           <string>过程块</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ActiveLabel" name="label_create_div">
          <property name="text">
           <string>分歧块</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>68</width>
         <height>450</height>
        </rect>
       </property>
       <attribute name="label">
        <string>高级</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ActiveView" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid wheat;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>244</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>made by zwh</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_run"/>
   <addaction name="action_step"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="action_tool">
   <property name="text">
    <string>工具</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_step">
   <property name="text">
    <string>步进</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActiveLabel</class>
   <extends>QLabel</extends>
   <header>ui/activelabel.h</header>
  </customwidget>
  <customwidget>
   <class>ActiveView</class>
   <extends>QWidget</extends>
   <header>ui/activeview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BlockScript.qrc"/>
 </resources>
 <connections/>
</ui>
